src/main/frontend/pipeline-console-view/pipeline-console/main/PipelineConsole.tsx

```diff
@@ -123,10 +123,15 @@ export default class PipelineConsole extends React.Component<
 
     // set default values of state
     this.state = {
+      // TODO - figure out why selectedStage and openStage are different?
```

Comment:
Why are `selectedStage` and `openStage` different properties? Surely the selected stage is the open stage and vice versa.

Reply:
Any ideas @timja?

Reply:
Still not sure why they're separate - but can address/tidy that in a separate PR.